.travis.yml

```diff
@@ -18,6 +18,10 @@ go:
 before_install:
   - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-6 90
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 90
+  - . $HOME/.nvm/nvm.sh
```

Comment:
ouch... we're making the CI slower because of the web :(
I didn't realize about of this during the design meeting :man_facepalming:

Reply:
🤷‍♀️